doc/lua_api.md

```diff
@@ -8477,8 +8477,17 @@ child will follow movement and rotation of that bone.
     * Passing no arguments resets lighting to its default values.
     * `light_definition` is a table with the following optional fields:
       * `saturation` sets the saturation (vividness; default: `1.0`).
-        * values > 1 increase the saturation
-        * values in [0,1] decrease the saturation
+        * It is applied according to the function `result = b*(1-s) + c*s`, where:
+          * `c` is the original color
+          * `b` is the greyscale version of the color with the same luma
+          * `s` is the saturation set here
+        * The resulting color always has the same luma (perceived brightness) as the original.
+        * This means that:
+          * value of 1 (default) doesn't change the saturation
+          * values > 1 increase the saturation
+          * values in [0,1] decrease the saturation
```

Comment:
nitpick: 0, 1 are exclusive. not sure which notation we want to use for this. `(0, 1)` might be misunderstood as a tuple, `]0, 1[` is ugly and somewhat uncommon, `> 0 and < 1` is a bit verbose.

Reply:
Changed that to "values between 0 and 1". Will be clear at least. Clearer than `> 0 and < 1` IMHO, even.